tox.ini

```diff
@@ -1,5 +1,16 @@
 [tox]
-envlist = py26, py27, py32, py33, py34
+envlist = py26, py27, py32, py33, py34, flake, flake_docs
 
 [testenv]
 commands = python tests.py
+
+[testenv:flake]
+deps = flake8
+exclude = docs/conf.py
+doctests = no
+
+[testenv:flake_docs]
+deps = flake8
+exclude = docs/conf.py
+doctests = yes
+ignore = F821
```

Comment:
Now that I look at these `[testenv:flake]` sections, aren't they missing something?  `commands = flake8 *.py`?  Also, flake8 isn't going to pick its settings out of `[testenv:...]` sections, it only looks for `flake8`.  I hope you can ignore a specific warning from the command line.

Maybe let's forget about applying flake8 checks to doctests?  I usually enable that because I often have top-level imports for names used only inside doctests, and adding `doctests = yes` to `[flake8]` in `setup.cfg` shuts off false-positives about those names being unused.